exampleCourse/questions/elementOverlayRandom/info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "uuid": "e93af989-ba82-4859-9b86-f850152341ce",
+    "title": "Element pl-overlay: Arbitrary positioning of existing elements",
```

Comment:
Can we rename this question to a `demoXXXXX` question? Maybe `demoTriangle`? And change the title to `Demo: randomize a drawing with overlaid inputs` or something like that?

I think we are aiming to have one `elementXXX` question per element, showing basic functionality. And then other more complex questions are `demoXXXX`.